OBSDOCS-217-Logging 5.6.7 on 4.10 (#60919)
openshift/openshift-docs: modules/logging-rn-5.6.7.adoc

// Module included in the following assemblies:
// cluster-logging-release-notes.adoc
:_content-type: REFERENCE
[id="cluster-logging-release-notes-5-6-7_{context}"]
= Logging 5.6.7
This release includes link:https://access.redhat.com/errata/RHBA-2023:3493[OpenShift Logging Bug Fix Release 5.6.7].

[id="openshift-logging-5-6-7-bug-fixes_{context}"]
== Bug fixes

> * Before this update, the LokiStack gateway returned label values for namespaces without applying the access rights of a user. With this update, the LokiStack-gateway applies permissions to label value requests, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3728[LOG-3728])

Reviewer: "LokiStack gateway" or "LokiStack-gateway"? I'd pick one for the same phrase.

> * Before this update, the `time` field of log messages did not parse as `structured.time` by default in Fluentd when they included a timestamp. With this update, parsed log messages will include a `structured.time` field if the output destination supports it. (link:https://issues.redhat.com/browse/LOG-4090[LOG-4090])

Reviewer: I'd also suggest "when the messages included a timestamp" instead of using "they"

> * Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to timeout. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])

Reviewer 2: ```suggestion
* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```

> * Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])

Reviewer 3: ```suggestion
* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affect the route timeout settings, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```

> * Before this update, LokiStack CRs with values defined for tenant limits but not global limits caused the Loki Operator to crash. With this update, the Operator is able to process LokiStack CRs with only tenant limits defined , resolving the issue. (link:https://issues.redhat.com/browse/LOG-4199[LOG-4199])

Reviewer 2: ```suggestion
* Before this update, LokiStack CRs with values defined for tenant limits but not global limits caused the Loki Operator to crash. With this update, the Operator is able to process LokiStack CRs with only tenant limits defined, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4199[LOG-4199])
```

* Before this update, the {product-title} web console generated errors after an upgrade due to cached files of the prior version retained by the web browser. With this update, these files are no longer cached, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4099[LOG-4099])

> * Before this update, Vector generated certificate errors when forwarding to default Loki. With this update, logs can be forwarded to Loki using Vector without errors. (link:https://issues.redhat.com/browse/LOG-4184[LOG-4184])

Reviewer 2: ```suggestion
* Before this update, Vector generated certificate errors when forwarding to the default Loki instance. With this update, logs can be forwarded without errors to Loki by using Vector. (link:https://issues.redhat.com/browse/LOG-4184[LOG-4184])
```

> * Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API will no longer require a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:

Reviewer 2: ```suggestion
* Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```

> * Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:

Reviewer 3: ```suggestion
* Before this update, the Cluster Logging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the Cluster Logging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```
